lib/commands/create/course.js

```diff
@@ -52,31 +52,25 @@ function deleteExistingCourse(properties) {
             if(exists) {
                 var deferred = Q.defer();
 
-                prompt.start();
-                prompt.get([{
+                inquirer.prompt([
+                    {
                         name: 'overwrite existing course?',
-                        message: 'Please specify (y)es or (n)o',
-                        pattern: /^y$|^n$/i,
-                        type: 'string',
-                        default: 'n',
-                        required: true,
-                        before: function(value) { return /^y$/i.test(value); }
-                    }],
-                    function (err, results) {
-                        if(err) deferred.reject(err);
-
-                        if(results['overwrite existing course?']) {
-                            fs.removeTree(properties.localDir)
-                                .then(function (){
-                                    deferred.resolve(properties);
-                                })
-                                .fail(function (err) {
-                                    deferred.reject(err);
-                                });
-                        } else {
-                            deferred.reject(new Error('Course already exists and cannot overwrite.'));
-                        }
-                    });
+                        type: 'confirm',
+                        default: false
+                    }
+                ]).then(results => {
+                    if(results['overwrite existing course?']) {
```

Comment:
White space

Reply:
Referring specifically to the lack of it between the `if` and `(`?

If I'm honest, I tried my hardest to ignore the existing code and linting/indentation/refactoring because the whole thing needs some TLC(!)

Reply:
Yes